utils/qtrunefu/qtefu.py

```diff
@@ -30,6 +30,24 @@ def _load(self, group, opt):
             print("No option {}, exiting ...".format(opt))
             sys.exit(0)
 
+    #  updates the [qtefu_latest] section of the ~/.efucfg file
+    def update_latest(self, selection):
+        for option in selection.keys():
```

Comment:
Something like `for name, value in selection.items()`
Then on line 36 `self.config.set("qtefu_latest", name, value)`